### PR TITLE
fix: sfdx create project (with manifest) to sf

### PR DESCRIPTION
**Pull request by CristiCanizales**
### What does this PR do?
- Updates project generate commands and flags to SF style in order to get rid of cli warnings

### What issues does this PR fix or reference?
@W-14720780@

### Functionality Before
- project generate commands in sfdx style

### Functionality After
- project generate commands in sf style

### CODE REVIEW

File: docs/_articles/en/lwc/linting.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Linting provides errors about malformed code while you edit. VS Code enforces Sa
 
 ### For a New Project
 
-If you create an SFDX project using the `sfdx force:project:create` command, your project contains a `package.json` file with the ESLint plugin already included.
+If you create an SFDX project using the `sfdx project:generate` command, your project contains a `package.json` file with the ESLint plugin already included.
```
Comment (peternhale) [inline docs/_articles/en/lwc/linting.md:16]: @CristiCanizales should `sfdx project:generate` be `sf project:generate`?

File: docs/_articles/en/source-control/source-format.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Follow these steps to convert the project from metadata to source format, withou
 
 1. Create a temporary SFDX project outside of the Git repo. This temporary project has the structure and a configuration file as required by a Salesforce project.
 
-   `$ sfdx force:project:create -n tempproj`
+   `$ sfdx project:generate -n tempproj`
```
Comment (peternhale) [inline docs/_articles/en/source-control/source-format.md:44]: Use `sf` command

File: docs/_articles/ja/lwc/linting.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ SSL のサポートを含めてビルドされた Node.js の [Active LTS リリ
 
 ### 新しいプロジェクト
 
-SFDX プロジェクトを、`sfdx force:project:create (SFDX: プロジェクトを作成)` コマンドで作成した場合、そのプロジェクトに含まれる `package.json` ファイルにはすでに ESLint のプラグインが含まれています。
+SFDX プロジェクトを、`sfdx project:generate (SFDX: プロジェクトを作成)` コマンドで作成した場合、そのプロジェクトに含まれる `package.json` ファイルにはすでに ESLint のプラグインが含まれています。
```
Comment (peternhale) [inline docs/_articles/ja/lwc/linting.md:16]: Same here

File: docs/_articles/ja/source-control/source-format.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ Force.com IDE ではメタデータ形式を使用していたため、VS Code 
 
 1. Git リポジトリの外で一時的な SFDX プロジェクトを作成します。この一時的なプロジェクトには Salesforce のプロジェクトで必要とされるディレクトリ構造や設定ファイルがあります。
 
-   `$ sfdx force:project:create -n tempproj`
+   `$ sfdx project:generate -n tempproj`
```
Comment (peternhale) [inline docs/_articles/ja/source-control/source-format.md:44]: And here

File: packages/salesforcedx-vscode-core/src/commands/projectGenerate.ts [hunk 5]
```diff
@@ -44,18 +44,16 @@ export class ProjectTemplateItem implements vscode.QuickPickItem {
...
 
   public constructor(options = { isProjectWithManifest: false }) {
     super();
     this.options = options;
   }
 
-  public executionName = nls.localize('force_project_create_text');
-  public telemetryName = 'force_project_create';
+  public executionName = nls.localize('project_generate_text');
+  public telemetryName = 'project_generate';
```
Comment (peternhale) [inline packages/salesforcedx-vscode-core/src/commands/projectGenerate.ts:56]: @CristiCanizales renaming the telemetry key for each of these command will break continuity in the telemetry data. I don't mind changing the name, just raising the consequence of doing so. Maybe add a section in the transition doc regarding this so folks can discuss?

**Review by peternhale — COMMENTED** [review]
@CristiCanizales a few questions for you

File: packages/salesforcedx-vscode-core/src/commands/projectGenerate.ts (hunk 5 above)
Comment (CristiCanizales) [reply packages/salesforcedx-vscode-core/src/commands/projectGenerate.ts:56]: Sure! I'll keep it as it is for now and after the discussion I'll create a WI if needed to update all telemetry keys 👍

File: docs/_articles/en/lwc/linting.md (hunk 1 above)
Comment (peternhale) [reply docs/_articles/en/lwc/linting.md]: If you would prefer to attend to the migration using `sfdx` and then revisit with a separate work item to address command name changes, I am ok with that.

File: packages/salesforcedx-vscode-core/test/vscode-integration/commands/projectGenerate.test.ts [hunk 6]
```diff
@@ -18,24 +18,24 @@ import * as vscode from 'vscode';
...
-  forceProjectWithManifestCreate,
-  forceSfdxProjectCreate,
   PathExistsChecker,
   ProjectNameAndPathAndTemplate,
-  projectTemplateEnum,
   ProjectTemplateItem,
   SelectProjectFolder,
   SelectProjectName,
-  SelectProjectTemplate
+  SelectProjectTemplate,
```
Comment (daphne-sfdc) [inline packages/salesforcedx-vscode-core/test/vscode-integration/commands/projectGenerate.test.ts:26]: Not in alphabetical order

**Review by daphne-sfdc — APPROVED** [review]
Approved! 🎉

Commands Tested:
1. SFDX: Create Project (Standard) ✅
2. SFDX: Create Project (Empty) ✅
3. SFDX: Create Project (Analytics) ✅
4. SFDX: Create Project with Manifest (Standard) ✅
5. SFDX: Create Project with Manifest (Empty) ✅
6. SFDX: Create Project with Manifest (Analytics) ✅

**Review by sbudhirajadoc — APPROVED** [review]

**Pull request merged**